Lib/defconAppKit/controls/fontInfoView.py

```diff
@@ -132,13 +132,15 @@ def _numberToString(self, value):
         if self._allowFloat:
             if not float(value).is_integer():
                 return self._floatFormat % value
-        return str(value)
+        return "%i" % value
 
     def _stringToNumber(self, string):
         value = None
         newString = string
         try:
             value = self._numberClass(string)
+            if self._allowFloat and float(value).is_integer():
```

Comment:
The `float()` cast isn't needed, as `self._allowFloat` being True already ensures `value` is a float.

Reply:
changed!